rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -2840,10 +2837,7 @@ namespace rsx
 				break;
 			}
 
-			vm::ptr<CellGcmReportData> out = sink;
-			out->value = value;
-			out->timer = timestamp;
-			out->padding = 0;
+			vm::_ref<atomic_t<CellGcmReportData>>(sink).store({ timestamp, value, 0});
```

Comment:
I would argue making this atomic is overkill, but I guess it doesn't matter much. Afaik Cell bus width is 64-bit so I would have expected no such guarantees in hardware. InterlockedExchange shouldn't be too bad I guess.